internal/cli/commands.go

```diff
@@ -171,6 +181,61 @@ func (h *CLICommands) outputDurationOrTimestamp(t time.Time) string {
 	return t.Format(time.RFC3339)
 }
 
+func (h *CLICommands) cmdGroupNodes() error {
+	params := url.Values{}
+	params.Add("group-name", h.groupName)
+	b, err := ReadFromURL("http://" + *h.ServerAddr + "/groups/nodes?" + params.Encode())
+	if err != nil {
+		return err
+	}
+
+	if h.outputFormat == formatJSON {
+		fmt.Printf("%s", string(b))
+		return nil
+	}
+
+	var result []diagnostics.NodeDiagnostics
+	if err := json.Unmarshal(b, &result); err != nil {
+		return err
+	}
+
+	table := table.NewTable([]string{
+		"Node", "ns", "ng", "zone", "taint", "filteredOut", "retry", "retry_after", "conditions", "stabilityPeriod", "canDrain",
```

Comment:
Just unifying the naming
```suggestion
		"Node", "NS", "NG", "Zone", "Taint", "Filtered Out", "Retry", "Retry After", "Conditions", "Stability Period", "canDrain",
```

Reply:
... and not `canDrain` 🤣 
Done.